schemas/rule.yml

```diff
@@ -81,4 +81,22 @@
 
       example: https://en.wikipedia.org/wiki/DNS_over_TLS
 
+    - name: author
+      level: extended
+      type: keyword
+      short: Rule author
+      description: >
+        Name, organization, or pseudonym of the author or authors who created the rule used to generate this event.
+
+      example: Start-Lord
+      normalize:
+        - array
+
+    - name: license_type
```

Comment:
I just noticed the field name `license_type` here. I think we should simply name this `license`, in order to be aligned with `package.license`.